src/test/java/blackjack/domain/state/BustTest.java

```diff
@@ -0,0 +1,62 @@
+package blackjack.domain.state;
+
+import static blackjack.controller.BlackJackController.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardNumber;
+import blackjack.domain.card.CardPattern;
+import blackjack.domain.card.Cards;
+import blackjack.domain.participant.Dealer;
+import blackjack.domain.participant.Money;
+
+class BustTest {
+	private PlayerState state;
+
+	@BeforeEach
+	void setUp() {
+		Cards cards = new Cards();
+		state = new Bust(cards);
+	}
+
+	@Test
+	@DisplayName("끝난 상태인지 확인")
+	void checkFinished() {
+		assertTrue(state.isFinished());
+	}
+
+	@Test
+	@DisplayName("Bust 맞음 확인")
+	void isBust() {
+		assertTrue(state.isBust());
+	}
+
+	@Test
+	@DisplayName("Blackjack 아님 확인")
+	void isNotBlackJack() {
+		assertFalse(state.isBlackJack());
+	}
+
+	@Test
+	@DisplayName("Bust 수익 계산")
+	void calculateDrawProfit() {
+		Dealer dealer = new Dealer();
+		dealer.receiveCard(new Card(CardPattern.DIAMOND, CardNumber.SEVEN));
+		dealer.receiveCard(new Card(CardPattern.DIAMOND, CardNumber.SIX));
+		state = state.keepContinue(false);
+		assertEquals(-100, state.makeProfit(dealer, new Money(100)));
+	}
+
+	@Test
+	@DisplayName("끝난 상태인데 새 카드 받으려 하면 에러 호출")
+	void exception() {
+		assertThatThrownBy(() -> state.drawNewCard(new Card(CardPattern.DIAMOND, CardNumber.NINE)))
```

Comment:
```suggestion
		Card card = new Card(CardPattern.DIAMOND, CardNumber.NINE);
		assertThatThrownBy(() -> state.drawNewCard(card))
```
현재 Card에서 에러가 안나긴하지만 확실하게 에러가 나는 곳을 구분하기 위해, `assertThatThrownBy(..)` 밖에서 선언해주세요~

Reply:
아 그럴 수 있다는걸 생각치 못했네요,,, 전부 수정했습니다!